addons/website_slides/models/slide_channel.py

```diff
@@ -231,9 +290,14 @@ def _get_default_enroll_msg(self):
         'res.partner', 'slide_channel_partner', 'channel_id', 'partner_id',
         string='Members', help="All members of the channel.", context={'active_test': False}, copy=False, depends=['channel_partner_ids'])
     members_count = fields.Integer('Attendees count', compute='_compute_members_count')
+    members_pending_count = fields.Integer('Attendees Pending count', compute='_compute_members_pending_count')
```

Comment:
I also renamed members_ongoing_count to members_joined_or_ongoing? This is useful in kanban view but actually covers both 'ongoing' and 'joined' members, as per spec. I could also change filter string / kanban view button string to 'Not Finished' instead of 'Ongoing', or leave it as such? (and action to action_redirect_to_joined_or_ongoing_members...) What do you think?

Reply:
Maybe we could find a word that covers both to avoid having long variable / method names?
What about "members_active_count" and "action_redirect_to_active_members" ?

Reply:
Sounds good, I will put the definition in the help field.